packages/js/tsup.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig, Options } from 'tsup';
+
+const baseConfig: Options = {
+  splitting: false,
+  sourcemap: false,
+  clean: true,
+  dts: true,
+};
+
+export default defineConfig([
+  { ...baseConfig, entry: ['src/index.ts'], format: ['esm', 'cjs'] },
+  {
+    ...baseConfig,
+    entry: { novu: 'src/umd.ts' },
+    format: ['iife'],
+    minify: true,
+    dts: false,
+    outExtension: () => {
+      return {
+        js: '.min.js',
+      };
+    },
```

Comment:
The bundled and minified JS file `novu.min.js` with IIFE that exposes the `Novu` to the `window` object